index.d.ts

```diff
@@ -46,7 +46,7 @@ declare namespace suitest {
 		setAppConfig(configId: string, options?: ConfigOverride): Promise<void|SuitestError>;
 		pairDevice(deviceId: string): Promise<DeviceData|SuitestError>;
 		releaseDevice(): Promise<void|SuitestError>;
-		startTest(clientTestId: string, options?: StartTestOptions): Promise<void|SuitestError>;
+		startTest(_?: string, __?: any): Promise<void|SuitestError>;
```

Comment:
I would say to user that startTest with arguments are deprecated, for example:
`/* @deprecated use startTest instead */
startTest(clientTestId: string, options?: StartTestOptions): Promise<void|SuitestError>;
startTest(): PromiseLike<void|SuitestError>`